### ISSUE TITLE
Sanitize sequence headers

### ISSUE THREAD

**Issue by ArtPoon** [PoonLab/covizu#63]
For example, I just noticed that this header is garbled with an extra forward slash:
```
hCoV-19//Estonia/ChVir4266/2020|EPI_ISL_457722|2020-03
```
Need to add a sanitizing step in `filtering.py`

**Comment by ArtPoon** [comment]
Header must conform to the following regex:
`^[^/]+/[A-Z][a-z -]+/[^/]+/[0-9]+\|[^|]+\|[0-9]{4}-[0-9]{2}-[0-9]{2}`

**Comment by ArtPoon** [comment]
This is harder than it looks.  Here are some examples of headers that are screened out by the following regex:
```regex
^hCoV-19/[A-Z][^/]+/[^/]+/[0-9]+\|[^|]+\|[0-9]{4}-[0-9]{2}-[0-9]{2}
```

* `hCoV-19/Germany/COV7/2020-03-26|EPI_ISL_457750|2020-03-26` - fill date in fourth field
* `hCov-19/Oman/RESP-20-4153/2020|EPI_ISL_457988|2020-03-21` - lower case `v`
* `hCoV-19/Belgium/ULG-6839/2020|EPI_ISL_447120|2020-3-14` - that's not how [ISO 8601](https://www.iso.org/iso-8601-date-and-time-format.html) works!

**Comment by ArtPoon** [comment]
Here's another one that's missing the year field in the first section:
`hCov-19/Bangladesh/DU_CARS_5586|EPI_ISL_450841|2020-05-13`

**Comment by ArtPoon** [comment]
This has an extra space:
`hCoV-19/USA/LA-EVTL08/2020 |EPI_ISL_435041|2020-04-15`

Some of these can be recovered.  Need to write a function for this in `filtering.py`

**Comment by ArtPoon** [comment]
Link to database issue? See #18

**Comment by ArtPoon** [comment]
Low priority - a small number of sequences are affected.

**Comment by ewong347** [comment]
Bad sequence date addressed with commit that closed #41.
Leaving this open for now, in case weird abnormalities in naming popup in the future.
